content/en/docs/contribute/blog/submission.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Submitting articles to Kubernetes blogs
+slug: article-submission
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+There are two official Kubernetes blogs, and the CNCF has its own blog where you can cover Kubernetes too.
+For the [main Kubernetes blog](/docs/contribute/blog/), we (the Kubernetes project) like to publish articles with different perspectives and special focuses, that have a link to Kubernetes.
+
+With only a few special case exceptions, we only publish content that hasn't been submitted or published anywhere else.
+
+<!-- body -->
+
+## Writing for the Kubernetes blog(s)
+
+As an author, you have three different routes towards publication.
+
+### Recommended route {#route-1}
+
+The approach the Kubernetes project recommends is: pitch your article by contacting the blog team. You can do that via Kubernetes Slack ([#sig-docs-blog](https://kubernetes.slack.com/archives/CJDHVD54J)).
+For articles that you want to publish to the contributor blog only, you can also pitch directly
+to [SIG ContribEx comms](https://kubernetes.slack.com/archives/C03KT3SUJ20).
+<!-- FIXME: or using this [form] -->
+
+Unless there's a problem with your submission, the blog team / SIG ContribEx will pair you up with:
```

Comment:
We have a a similar walk-through of the process [here](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/blogging-resources/sig-spotlight-blog-process.md), when this is merged I will open a PR to update it, referencing this one and focusing on the things that are specific to spotlights.